js/deer-config.js

```diff
@@ -189,6 +189,7 @@ export default {
                                 }
                             }
                         })
+                        if(!filteringProps.includes("title")) li += `data-title="[ unlabeled ]"`
```

Comment:
Is it better to hardcode this value everywhere or just set it empty and handle it being empty in the rendering?